packages/cli/src/commands/update.ts

```diff
@@ -0,0 +1,176 @@
+import fetch from "node-fetch";
+import inquirer from "inquirer";
+import semver from "semver";
+import { readFileSync } from 'fs';
+import { logger } from "../utils/logger.js";
+import { getUserPkgManager } from "../utils/getUserPkgManager.js";
+import { exec } from "child_process";
+import { installDependencies } from "../utils/installDependencies.js";
+import { resolvePath } from "../utils/parseNameAndPath.js";
+import { writeJSONFile } from "../utils/fileSystem.js";
+
+export async function updateCommand(path: string) {
+    try {
+        const resolvedPath = resolvePath(path);
+        const installedPackages = await getInstalledPackages(resolvedPath);
+        const triggerDevPackagesName = Object.keys(installedPackages.dependencies).filter(pkg => pkg.startsWith('@trigger.dev/'));
+
+        if (triggerDevPackagesName.length === 0) {
+            logger.info('No @trigger.dev packages found in package.json.');
+            return;
+        }
+
+        const triggerDevPackagesVersions = Object.fromEntries(triggerDevPackagesName.map(packageName => {
+            const currentVersion = installedPackages.dependencies[packageName].version;
+            return [packageName, currentVersion];
+        }));
+
+        const latestVersions = await getLatestVersions(triggerDevPackagesName, triggerDevPackagesVersions);
+        const updates: { [packageName: string]: string } = {};
+
+        triggerDevPackagesName.forEach(packageName => {
+            const currentVersion = installedPackages.dependencies[packageName].version;
+            const latestVersion = latestVersions[packageName];
+            if (latestVersion) {
+                if (semver.gt(latestVersion, currentVersion)) {
+                    updates[packageName] = latestVersion;
+                }
+            }
+        });
+
+
+        if (Object.keys(updates).length === 0) {
+            console.log('No updates available.');
+            return;
+        }
+
+        const confirm = await confirmUpdate(updates);
+
+        if (confirm) {
+            const updatedPackageJson = updatePackageJson(updates);
+            await writeJSONFile('package.json', updatedPackageJson);
```

Comment:
This is not respecting the `resolvedPath` so the wrong `package.json` will get written to if `path` isn't `.`. Need to do:

```ts
import pathModule from "path";

await writeJSONFile(pathModule.join(resolvedPath, 'package.json'), updatedPackageJson);
```

Reply:
done